MAUI/ImageEditor/shape.md

```diff
@@ -51,6 +51,91 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Polygon
+
+A polygon is formed by connecting a series of straight lines, and in addition to the `Points` collection includes specific points, a line is automatically connects the first and last points. 
```

Comment:
A polygon is formed by connecting a series of straight lines, and in addition to the `Points` collection including specific points, a line automatically connects the first and last points.